packages/ui/src/Layouts/PageLayoutHeader.tsx

```diff
@@ -2,10 +2,16 @@
 import { ReactProps } from '@bloxifi/types'
 import classNames from 'classnames'
 import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
 
-import { Text } from '../Text'
+import { Fonts } from '../styles/fonts'
 
-import { BoxLayout } from './BoxLayout'
+type NavItemType = {
+  to: string
```

Comment:
We should have comments here and maybe switch to `interface NavigationItem` No need to add `Type` or `T` in type name.

[**_Interface work better with objects and method objects, and types are better to work with functions, complex types, etc._**](https://blog.logrocket.com/types-vs-interfaces-in-typescript/)

Reply:
Good to know! :)

Reply:
Updated! :)